src/docs/product/data-management-settings/server-side-sampling/getting-started.mdx

```diff
@@ -0,0 +1,81 @@
+
+</Note>
+
+<Note>
+
+To set up or configure server-side sampling, you need to be an Owner, Manager, or Admin of your Sentry organization.
+
+</Note>
+
+## Current Limitations
+
+Server-side sampling is currently only available for the following SDK's:
+
+- Python: 1.7.2 or later
+- JavaScript: 7.6.0 or later
+- Apple: 7.23.0 or later
+- Android: 6.2.1 or later
+
+If your application relies on any Sentry SDK that isn't specified above, then you won’t be able to use server-side sampling.
+
+There are other limitations that you should be aware of when setting up server-side sampling:
+
+- [Trace rules can only select based on the attributes of the initial transaction](/product/data-management-settings/server-side-sampling/current-limitations/#trace-rules-can-only-select-based-on-the-attributes-of-the-initial-transaction)
+- [Changing trace attributes in secondary transactions](/product/data-management-settings/server-side-sampling/current-limitations/#changing-trace-attributes-in-secondary-transactions)
+- [Limited number of rule conditions](/product/data-management-settings/server-side-sampling/current-limitations/#limited-number-of-rule-conditions)
+- [Arbitrary logical composition for rule conditions](/product/data-management-settings/server-side-sampling/current-limitations/#arbitrary-logical-composition-for-rule-conditions)
+
+For more detailed information about each of these, check out our [full Current Limitations documentation](/product/data-management-settings/server-side-sampling/current-limitations/).
+
+## 1. Update Sentry SDK Versions
+
+Once you’ve confirmed that your application only relies on [supported SDKs](#current-limitations), then the next step is to update them to the specified version or higher. Check out the migration documentation for the [SDKs](/platforms/) you’re using to learn everything you need to know to get up and running again with the latest Sentry features.
+
+## 2. Set a Uniform Sampling Rate
+
+Next, go to **[Project] > Settings > Server-side Sampling** in [sentry.io](https://sentry.io) and set a uniform sampling rate for all transactions: this is the number of transactions that Sentry should accept from your application:
+
+![Setting a uniform sampling rate for your project](set-uniform-sampling-rate.png)
+
+Based on your quota and usage, in this step, we’ll suggest sampling rates for both Sentry servers and your Sentry SDK. However, you can enter a custom sampling rate by overriding our recommended values. For example, you can select your current sampling rates to understand how our proposed sampling configuration would impact your transaction volume within the scope of this project. Learn more about [How Sentry Samples](/product/data-management-settings/server-side-sampling/#how-sentry-samples).
```

Comment:
> Based on your quota and usage, in this step, we’ll suggest sampling rates for both Sentry servers and your Sentry SDK

"Sentry SDK" sounds strange to me... does it mean only the SDKs supported by us?

Reply:
your in this case refers to the Sentry SDK your project uses to set the sampling rate.